source/UF/Size-TruncatedConnected.lagda

```diff
@@ -0,0 +1,423 @@
+     cod-con = trunc-map-is-connected ∣ x' ∣[ n + 1 ]
+     f-con : f is n connected-map
+     f-con = basepoint-map-is-less-connected (ua _) f cod-con
+     𝟙-is-small : 𝟙 {𝓤} is 𝓥 small
+     𝟙-is-small = (𝟙 {𝓥} , one-𝟙-only)
+     cod-locally-small : (Σ x ꞉ X , ∣ x ∣[ n + 1 ] ＝ ∣ x' ∣[ n + 1 ])
+                          is ι (n + 2) locally-small
+     cod-locally-small = local-smallness-is-closed-under-Σ X-loc-small
+                          path-locally-small
+      where
+       path-locally-small : (x : X)
+                          → (∣ x ∣[ succ n ] ＝ ∣ x' ∣[ succ n ])
+                             is ι (n + 2) locally-small
+       path-locally-small x =
+        local-smallness-is-closed-under-≃ (eliminated-trunc-identity-char (ua _))
+                                     path-char-locally-small
+         where
+          path-char-locally-small : ∥ x ＝ x' ∥[ n ] is ι (n + 2) locally-small
+          path-char-locally-small =
+           local-smallness-is-closed-under-truncation
+            (being-locally-small-is-upper-closed X-loc-small x x')
+   fiber-path-space-small : (y : ∥ X ∥[ n + 1 ])
+                          → (Σ x ꞉ X , ∣ x ∣[ n + 1 ] ＝ y) is 𝓥 small
+   fiber-path-space-small =
+    ∥∥ₙ-ind (λ - → props-are-truncated pt
+                    (being-small-is-prop ua (Σ x ꞉ X , ∣ x ∣[ n + 1 ] ＝ -) 𝓥))
+            trunc-ind-helper
+
+ small-from-locally-and-small-truncation : {X : 𝓤 ̇} {n : ℕ₋₂}
+                                         → Join-Construction-Result {𝓤} {𝓤}
+                                         → X is ι (n + 2) locally-small
+                                          × ∥ X ∥[ n + 1 ] is 𝓥 small
+                                         → X is 𝓥 small
+ small-from-locally-and-small-truncation {_} {X} {n} j small-hyp =
+  smallness-closed-under-≃' (small-path-space-lemma j small-hyp)
+                            (domain-is-total-fiber ∣_∣[ succ n ])
+
+\end{code}
+
+Theorem 2.6. of [1]
+
+\begin{code}
+
+ Theorem-2-6[type-is-small-iff-type-is-locally-small-and-has-small-truncation]
+  : {X : 𝓤 ̇} {n : ℕ₋₂}
+  → Join-Construction-Result {𝓤} {𝓤}
+  → X is 𝓥 small
+  ↔ X is ι (n + 2) locally-small × ∥ X ∥[ n + 1 ] is 𝓥 small 
+ Theorem-2-6[type-is-small-iff-type-is-locally-small-and-has-small-truncation]
+  {_} {X} {n} j =
+  (foreward , small-from-locally-and-small-truncation j)
+  where
+   foreward : X is 𝓥 small
+            → X is ι (n + 2) locally-small × ∥ X ∥[ n + 1 ] is 𝓥 small
+   foreward X-small =
+    (small-gives-locally-small X-small ,
+     truncations-of-small-types-are-small X-small)
+
+ type-is-small-iff-type-is-locally-small-and-has-small-truncation =
+  Theorem-2-6[type-is-small-iff-type-is-locally-small-and-has-small-truncation]
+
+\end{code}
+
+We will record the following corolloary of Theorem 2.6. of [1]:
+ The set truncation of a universe is large (i.e. not small).
+
+\begin{code}
+
+ set-truncation-of-universe-is-large : Join-Construction-Result 
```

Comment:
You could just conclude that `is-large (𝓥 ̇ )`, right? (And why not use U rather than V?)

Reply:
We already have that result: ```universes-are-large : is-large (𝓤 ̇ )
universes-are-large = II``` 
unless I am misunderstanding the suggestion.